<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-MHRK6LP" failures="4" tests="4" name="Test(failed)" time="0.128" errors="0" timestamp="2022-04-21T11:29:45 IST">
  <testcase classname="TestNG.assertion06" name="assertnotnull" time="0.02">
    <failure type="java.lang.AssertionError" message="expected object to not be null">
      <![CDATA[java.lang.AssertionError: expected object to not be null
at TestNG.assertion06.assertnotnull(assertion06.java:59)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- assertnotnull -->
  <testcase classname="TestNG.assertion06" name="assertfalse" time="0.006">
    <failure type="java.lang.AssertionError" message="&amp;lt;falied 2&amp;gt; expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: <falied 2> expected [false] but found [true]
at TestNG.assertion06.assertfalse(assertion06.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- assertfalse -->
  <testcase classname="TestNG.assertion06" name="assernotequal" time="0.003">
    <failure type="java.lang.AssertionError" message="failed 2 cases expected [hi] but found [hello]">
      <![CDATA[java.lang.AssertionError: failed 2 cases expected [hi] but found [hello]
at TestNG.assertion06.assernotequal(assertion06.java:28)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- assernotequal -->
  <testcase classname="TestNG.assertion06" name="asserteql" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [hi] but found [hello]">
      <![CDATA[java.lang.AssertionError: expected [hi] but found [hello]
at TestNG.assertion06.asserteql(assertion06.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- asserteql -->
  <testcase classname="TestNG.assertion06" name="asserttru" time="0.0">
    <ignored/>
  </testcase> <!-- asserttru -->
  <testcase classname="TestNG.assertion06" name="assertnull" time="0.0">
    <ignored/>
  </testcase> <!-- assertnull -->
</testsuite> <!-- Test(failed) -->
